Tasks/ArchiveFiles/task.loc.json

```diff
@@ -16,8 +16,8 @@
   ],
   "demands": [],
   "version": {
-    "Major": 1,
-    "Minor": 119,
+    "Major": 2,
```

Comment:
Can we avoid renaming the input and just add an alias?

Reply:
It seems to me we have to update the major version so that we can change the `archiveType` default value to `zip`.  And if we change the major version, I figured we might as well change the input name to `rootFolderOrFile` rather than provide an alias so that the code is cleaner.  Do you disagree?

Reply:
I thought that might be the case :(.